modules/secure-logging/Makefile.am

```diff
@@ -0,0 +1,31 @@
+module_LTLIBRARIES					+=	\
+	modules/secure-logging/libsecure-logging.la
+
+EXTRA_DIST += modules/secure-logging/CMakeLists.txt
+
+modules_secure_logging_libsecure_logging_la_CFLAGS	=	\
+	-I$(top_srcdir)/modules/secure-logging	                \
```

Comment:
You are missing a `$(AM_CFLAGS)` here.

Reply:
Thank you very much. I would not have found this by myself, as everything build correctly when executling the ```make distcheck``` target manually on my build system. I corrected this problem with the latest commit to the forked repository and hope that the secure logging functionality will finally be ready for review.